enterprise/cypher/pom.xml

```diff
@@ -21,6 +21,8 @@
     <licensing.prepend.text>notice-agpl-prefix.txt</licensing.prepend.text>
     <license-text.header>AGPL-3-header.txt</license-text.header>
     <opencypher.version>1.0.0-M09</opencypher.version>
+    <junit-plattform-runner.version>1.1.0</junit-plattform-runner.version>
+    <junit-jupiter.version>5.1.0</junit-jupiter.version>
```

Comment:
also i would really like to avoid situation when different submodules that use different versions of libraries that should be common for whole project, just because they can